src/Cake.Npm/Install/NpmInstallSettingsExtensions.cs

```diff
@@ -0,0 +1,259 @@
+        /// <param name="settings">The settings.</param>
+        /// <param name="value">Value which should be set.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.Force"/> set to <c>false</c>.</returns>
+        public static NpmInstallSettings WithoutForce(this NpmInstallSettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            return settings.WithForce(false);
+        }
+
+        /// <summary>
+        /// Defines whether npm should fetch remote resources even if a local copy
+        /// exists on disk.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="value">Value which should be set.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.Force"/> set to <paramref name="value"/>.</returns>
+        public static NpmInstallSettings WithForce(this NpmInstallSettings settings, bool value)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            settings.Force = value;
+            return settings;
+        }
+
+        /// <summary>
+        /// Defines that the package should be installed globally.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.Global"/> set to <c>true</c>.</returns>
+        public static NpmInstallSettings InstallGlobally(this NpmInstallSettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            settings.Global = true;
+            return settings;
+        }
+
+        /// <summary>
+        /// Defines that the package should be installed locally.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.Global"/> set to <c>false</c>.</returns>
+        public static NpmInstallSettings InstallLocally(this NpmInstallSettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            settings.Global = false;
+            return settings;
+        }
+
+        /// <summary>
+        /// Defines that npm should ignore modules listed in <c>devDependencies</c>.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.Production"/> set to <c>true</c>.</returns>
+        public static NpmInstallSettings OnProduction(this NpmInstallSettings settings)
```

Comment:
Not convinced by the term `OnProduction`, perhaps something like `.ForProduction` or `.ProductionOnly` ?